tests/test-sources/plugins/utils/neocord.nix

```diff
@@ -14,7 +14,7 @@
       settings = {
         # General options.
         logo = "auto";
-        logo_tooltip = null;
+        logo_tooltip = "Nixvim";
```

Comment:
```suggestion
        logo_tooltip = null;
```

Reply:
You could have left the one in the `example` config.

Reply:
> You could have left the one in the `example` config.

Yeah, I should've done that. Thanks again for guiding me haha